scan/templates/ats/detail.html

```diff
@@ -11,15 +11,72 @@
 {% endblock %}
 
 {% block scripts %}
- <script>
-    var qrcode = new QRCode(document.getElementById("qr_code"), {
-        text: "{{ at.id|num2rs }}",
-        dotScale: .7,
-        backgroundImage: "/static/Signum_Logomark_blue_filled.svg",
-        backgroundImageAlpha: .3,
-        logoBackgroundTransparent: true,
-        correctLevel: QRCode.CorrectLevel.H,
-    }); 
+<script>
+  var qrcode = new QRCode(document.getElementById("qr_code"), {
+      text: "{{ at.id|num2rs }}",
+      dotScale: .7,
+      backgroundImage: "/static/Signum_Logomark_blue_filled.svg",
+      backgroundImageAlpha: .3,
+      logoBackgroundTransparent: true,
+      correctLevel: QRCode.CorrectLevel.H,
+  }); 
+</script>
+<script type="module">
+  import decompiler from 'https://cdn.jsdelivr.net/npm/smartc-signum-decompiler/dist/index.js';
```

Comment:
Can we add this locally please

Reply:
I'm not sure what the benefit of that would be since it would have to be manually updated when deleterium updated it. I also couldn't get it to work on a local import. Using load static should work, however it wouldn't run. This was a very amateur attempt to incorporate this, I don't know enough about js to troubleshoot it.

Reply:
It is more about data protection law from the EU.

Reply:
Understood, I'll figure it out tomorrow and patch that up. We might have to take a look at some other pages as well then as many have js imports that I'm not even entirely sure why they're there.

Reply:
yes thanks a lot

Reply:
Yeah... we are updating parts of the entire ecosystem, to have 3rd party libs not coming from CDNs... EU Law sucks.

Reply:
@frankTheTank72 @ohager 

Done, added locally and updated in PR